hex/README.md

```diff
@@ -0,0 +1,18 @@
+# Fast Hexadecimal Encoding/Decoding
+
+This is a hex library designed to be fast.
+
+## Minimum Supported Rust Version (MSRV)
+
+This library should always compile with any combination of features on **Rust 1.41.1**.
```

Comment:
"always" seems too strong. I definitely want to support fixed-sized arrays at some point with higher MSRV.

Reply:
Perhaps MSRV does not imply `--all-features` (or all combinations) but it should definitely always compile with some subset of features, right?

If this sentence is wrong it needs fixing organisation wide.

Reply:
Hmm, I guess I'm misinterpreting "always" as "forever". But yes, we should have exception for some features.

Reply:
Flagging this thread because I'm not 100% sure current (soon to be pushed) implementation resolves it.

Reply:
I think we should drop the word "always" if it makes Kix nervous. It's silly to claim any permanent rules for a 0.0.1 crate.

Reply:
I used:
```